sbom-utility-scripts/scripts/sbom-for-modelcar-task/sbom_for_modelcar_task.py

```diff
@@ -0,0 +1,261 @@
+"""
+This script provides a helper function to generate an SBOM reports for ModelCar images.
+
+More information about ModelCar images can be found at:
+ - https://developers.redhat.com/articles/2025/01/30/build-and-deploy-modelcar-container-openshift-ai#
+ - https://kserve.github.io/website/latest/modelserving/storage/oci/
+
+The script requires 2 references, the Model OCI artifact where the model files will be extracted from, and the Base Image for the ModelCar image.
+
+The SBOM report has the following structure:
+
+- There are 3 components reported in the SBOM, the Modelcar image, and the Base and Model images
+- The Modelcar component is a DESCENDANT OF both the Model and the Base images
+
+Usage:
+
+    sbom_for_modelcar_task.py [-h] --modelcar-image MODELCAR_IMAGE --base-image BASE_IMAGE --model-image MODEL_IMAGE [-o OUTPUT_FILE] [--sbom-type {cyclonedx,spdx}]
+
+Options:
+    -h, --help        Show this help message and exit
+    --modelcar-image  ModelCar OCI artifact reference resolved to digest (e.g., quay.io/foo/modelcar_image@sha256:abcdef1234567890...
+    --base-image      Base Image OCI artifact reference resolved to digest (e.g., quay.io/foo/base_image@sha256:abcdef1234567890...
+    --model-image     Model OCI artifact reference resolved to digest (e.g., quay.io/foo/model_image@sha256:abcdef1234567890...
+    --sbom-type       SBOM report format [CycloneDX, SPDX] - Optional, default: CycloneDX
+    -o, --output-file   Output file path
+"""
```

Comment:
nitpick: you don't need to write such a long docstring. Instead, using the argparse API to set program description and arguments description, then, `-h` will print the whole description.

Reply:
Thanks for the suggestion, it makes sense. I've accordingly updated the docstring to match the output of the help function, and refactored the former contents using the Argparse API as suggested.